auth/src/main/java/com/farsitel/bazaar/BazaarClientProxy.kt

```diff
@@ -1,17 +1,20 @@
-package com.farsitel.bazaar.auth
+package com.farsitel.bazaar
 
 import android.content.Context
 import com.farsitel.bazaar.auth.security.Security
-import com.farsitel.bazaar.auth.util.getPackageInfo
-import com.farsitel.bazaar.auth.util.versionCodeSDKAware
+import com.farsitel.bazaar.util.getPackageInfo
+import com.farsitel.bazaar.util.versionCodeSDKAware
 import com.farsitel.bazaar.auth.view.BazaarInstallerActivity
 
 object BazaarClientProxy {
 
     private const val BAZAAR_WITH_AUTH_VERSION = 801500
 
     fun isBazaarInstalledOnDevice(context: Context): Boolean {
-        return getPackageInfo(context, BAZAAR_PACKAGE_NAME) != null &&
+        return getPackageInfo(
+            context,
+            BAZAAR_PACKAGE_NAME
+        ) != null &&
                 Security.verifyBazaarIsInstalled(context)
```

Comment:
```suggestion
        return getPackageInfo(context, BAZAAR_PACKAGE_NAME) != null && 
            Security.verifyBazaarIsInstalled(context)
```
This looks better IMO.

Reply:
Done